sdk/messaging/azservicebus/README.md

```diff
@@ -0,0 +1,50 @@
+# Microsoft Azure Service Bus Client for Golang
+[![Go Report Card](https://goreportcard.com/badge/github.com/Azure/azure-service-bus-go)](https://goreportcard.com/report/github.com/Azure/azure-service-bus-go)
+[![godoc](https://godoc.org/github.com/Azure/azure-service-bus-go?status.svg)](https://godoc.org/github.com/Azure/azure-service-bus-go)
+[![Build Status](https://travis-ci.org/Azure/azure-service-bus-go.svg?branch=master)](https://travis-ci.org/Azure/azure-service-bus-go)
+[![Coverage Status](https://coveralls.io/repos/github/Azure/azure-service-bus-go/badge.svg?branch=master)](https://coveralls.io/github/Azure/azure-service-bus-go?branch=master)
+
+Microsoft Azure Service Bus is a reliable cloud messaging service (MaaS) which simplifies enterprise cloud messaging. It
+enables developers to build scalable cloud solutions and implement complex messaging workflows over an efficient binary
+protocol called AMQP.
+
+This library provides a simple interface for sending, receiving and managing Service Bus entities such as Queues, Topics
+and Subscriptions.
+
+For more information about Service Bus, check out the [Azure documentation](https://azure.microsoft.com/services/service-bus/).
+
+This library is a pure Golang implementation of Azure Service Bus over AMQP.
+
+## Preview of Service Bus for Golang
+This library is currently a preview. There may be breaking interface changes until it reaches semantic version `v1.0.0`. 
+If you run into an issue, please don't hesitate to log a 
+[new issue](https://github.com/Azure/azure-service-bus-go/issues/new) or open a pull request.
```

Comment:
This should change to the azure-sdk-for-go repo and tag with the appropriate tag for SB

Reply:
Done in https://github.com/Azure/azure-sdk-for-go/pull/15699